mrp_production_real_cost/README.rst

```diff
@@ -26,14 +26,15 @@ https://github.com/OCA/product-variant
 Usage
 =====
 
-Operating with a manufacture order, analytic entries adding costs will be
+Processing a manufacture order, analytic entries adding costs will be
 created when:
 
 * Some raw material is consumed.
 * A work order is finished or paused.
+* Also, together with *project_timesheet* module, users time is also translated
+to costs in the linked analytic account.
 
-Also, thanks to *project_timesheet* modules, users time is also translated to
```

Comment:
I chose mp_project because the relationship between `account_analytic_line` and `mrp_production` is created there. I was thinking to be close to that with these changes. 

But the costing issue I found was related to `mrp_production_real_cost` ... and that's where users are going to read the module description